### PR TITLE
NN-5478: remove /proven-adjudication endpoint

### PR DESCRIPTION
**Pull request by neilmendum**
https://dsdmoj.atlassian.net/browse/NN-5478

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java [hunk 1]
```diff
@@ -182,18 +181,6 @@ private int getAdjudicationCount(LocalDate awardCutoffDateParam, LocalDate adjud
         return adjudicationCount;
     }
 
-    public List<ProvenAdjudicationSummary> getProvenAdjudications(final List<Long> bookingIds,
-                                                                  final LocalDate cutoffDateParam) {
-        return repository.findAwardsForMultipleBookings(bookingIds).stream()
```
Comment (Mjwillis) [inline src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java:187]: repository.findAwardsForMultipleBookings can be removed

Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java:187]: FIND_AWARDS_BY_BOOKINGS enum can be removed too from `AdjudicationRepositorySql.kt`

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java [hunk 2]
```diff
@@ -889,55 +889,6 @@ public void getVisitsWithVisitorsFilteredByCancellationReason() {
...
-            final var httpEntity = createHttpEntity(token, List.of(-5,-8));
-
-            final var response = testRestTemplate.exchange(
-                "/api/bookings/proven-adjudications?adjudicationCutoffDate=2017-09-13",
-                HttpMethod.POST,
-                httpEntity,
-                new ParameterizedTypeReference<String>() {
-                });
-
-            assertThatJsonFileAndStatus(response, 200, "proven_adjudications.json");
```
Comment (Mjwillis) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java:937]: remove file

**Review by Mjwillis — CHANGES_REQUESTED** [review]
Three other codes bits can be removed

File: src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java (hunk 1 above)
Comment (neilmendum) [reply src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java:187]: good spot - addressed with https://github.com/ministryofjustice/prison-api/pull/1720/commits/003c1df263c73ffaf74f4870f0eb9f9418ce4508

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java (hunk 2 above)
Comment (neilmendum) [reply src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java:937]: good spot - addressed with https://github.com/ministryofjustice/prison-api/pull/1720/commits/003c1df263c73ffaf74f4870f0eb9f9418ce4508

**Review by Mjwillis — APPROVED** [review]
LGTM

**Comment by Mjwillis** [comment]
@neilmendum could we raise a separate PR to mark all these endpoints as deprecated with a comment to say what the new API endpoint is

**Pull request merged**